mgmt/Cargo.toml

```diff
@@ -11,7 +11,7 @@ path = "tests/reconcile.rs"
 required-features = ["bolero"]
 
 [features]
-default = ["bolero"]
```

Comment:
Does removing bolero here also mean that fuzz testing isn't in the default `cargo test` run?

Reply:
No.  bolero is also listed as a dev dependency.  It is always used by the tests.

This controls the optionality of bolero for produced artifacts.  Basically, do we want downstream librarires to be able to use the fuzzers from this crate.  The answer to that is "maybe, but only if they are doing a test build."

So it shouldn't be on by default

Reply:
No.  bolero is also listed as a dev dependency.  It is always used by the tests.

This controls the optionality of bolero for produced artifacts.  Basically, do we want downstream libraries to be able to use the fuzzers from this crate.  The answer to that is "maybe, but only if they are doing a test build."

So it shouldn't be on by default